src/Mattermost/MattermostExtendSocialite.php

```diff
@@ -4,7 +4,7 @@
 
 use SocialiteProviders\Manager\SocialiteWasCalled;
 
-class MatterMostExtendSocialite
+class MattermostExtendSocialite
```

Comment:
Guess this will need a major version bump as well?

Reply:
nope, it was actually the "style fixes" commit that changed it, i'm simply changing it back:
https://github.com/SocialiteProviders/Providers/commits/master/src/Mattermost/MatterMostExtendSocialite.php

Reply:
Ahh. good catch, I will check if any other ones were changed.